rotkehlchen/exchanges/iconomi.py

```diff
@@ -363,7 +364,7 @@ def query_supported_tickers(
         for asset_info in resp:
             if not asset_info['supported']:
                 continue
-            if asset_info['ticker'] in UNSUPPORTED_ICONOMI_ASSETS:
+            if asset_info['ticker'] in GlobalDBHandler.get_unsupported_assets(Location.ICONOMI):
```

Comment:
you are querying this in a for loop. Instead query the db once outside the loop and reference the set